cli/module_reload_test.go

```diff
@@ -166,11 +180,13 @@ func TestFullReloadFlow(t *testing.T) {
 				map[string]any{
 					moduleFlagPath: manifestPath, generalFlagPartID: "part-123",
 					moduleBuildFlagNoBuild: true, moduleFlagLocal: true,
+					generalFlagNoProgress: true, // Disable progress spinner to avoid race conditions in tests
```

Comment:
It seems the progress spinner is considered harmful. Did we identify what the race is?